Server/Python/src/dbs/business/DBSBlockInsert.py

```diff
@@ -254,8 +254,10 @@ def insertBlockFile(self, blockcontent, datasetId, migration=False):
                     hasFparentage = True
                     if migration:
                         fileParentList[k]['this_file_id'] = logicalFileName[fileParentList[k]['this_logical_file_name']]
-                        del fileParentList[k]['this_logical_file_name']
```

Comment:
I would suggest to delete these keys/values with a cleaner command, like:
```
fileParentList[k].pop("this_logical_file_name", None)
fileParentList[k].pop("parent_file_id", None)
```

Reply:
ok, thanks, I adjusted the code to your suggestion